README.md

```diff
@@ -219,6 +219,7 @@ FLAGS:
                                      The default behavior is to output a space character in place of these markers.
         --light                      Use default colors appropriate for a light terminal background. For more control,
                                      see the other color options.
+        --line-numbers               Show line numbers for each line of the diff hunk.
```

Comment:
This is out of date now. (I do this step with `delta --help >> README.md` and then moving the appended text into position and fiddling around a bit.)

Reply:
👍 done.